Resolving #2285 UI: Fixed renderer stats [manual reset].

TheoTheDev opened the pull request:
UI stats: Fixed number of DrawCalls in stats panel [when H is pressed].
Problem: Cause of Post-processing renderer was rendering frame with one draw call and 'reseting' internal stats info.
Fix: Fixed by manually reseting renderer info before each frame render.
<img width="908" alt="image" src="https://user-images.githubusercontent.com/1478866/161586292-4be47786-8cea-49ad-8e89-5685dd967342.png">

ps: There was missing skeleton update for head part of the character [and without this manual update it has issues].

alisaad673 commented:
This seems to be working fine. Previously, There's was only one draw call being shown in stats panel. Now its being continuously updated.

avaer reviewed (changes requested):
Extra code?

avaer commented:
> ps: There was missing skeleton update for head part of the character [and without this manual update it has issues].

This should be another PR.

TheoTheDev commented on metaversefile-api.js:
```diff
@@ -479,6 +479,7 @@ metaversefile.setApi({
       if (localPlayer.avatar) {
         wasDecapitated = localPlayer.avatar.decapitated;
         localPlayer.avatar.undecapitate();
+        localPlayer.avatar.skeleton.update();
```
cause if this will be missing skeleton won't be updated for second render in the mirror.
as a result players head will be missing in the reflection.

TheoTheDev replied:
Before you reverted same PR of other dev BECAUSE of the issue with missing head in the reflection.
Now you say it's ok to make this PR without this fix.
Plz be consistent in your requests, this can continue eternally.

avaer replied:
> Now you say it's ok to make this PR without this fix.

I didn't say that...

TheoTheDev replied:
so what exactly you suggest ?
this is required thing, maybe can be moved to .undecapitate method,
but same was in neighbour code about the opposite and same structure so I did same.

TheoTheDev replied:
So ?

avaer commented:
Looks like this PR has been abandoned. If there is progress, please re-open.

TheoTheDev commented:
There was no progress cause I was waiting for your reply.
In any case, I removed the fix for the missing head in the mirror in case of first-person mode.
Will create for this second PR as you requested.
As for now this PR fixes the draw calls number,
but brings the issue like this
![image](https://user-images.githubusercontent.com/1478866/163496336-0a0ae39f-cb4c-4078-a94b-d7dd86e12d4d.png)
Which was fixed by updating skeleton in the useBeforeRender function.

avaer commented:
Ok if this breaks the functionality then we can't ship it.

TheoTheDev commented:
Updated the Mirror app with renderer.info.render.frame ++
To ask the renderer to update skeleton.
Removed extra skeleton update in useAfterRender.
So now it's more performance optimized.
And all issues are resolved.

avaer commented:
No correct solution could be found by the author in THREE.js.

It needs a THREE.js expert.
